infra/app/database.bicep

```
@@ -60,6 +63,11 @@ var containers = [
           type: 'diskANN'
         }
       ]
+      fullTextIndexes: [
+        {
+          path: '/description',
```

Comma throwing syntax error

```suggestion
          path: '/description'
```